### ISSUE TITLE
Not able to start workloads using k8s version - 1.13.4

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#18527]
Rancher server version - v2.1.7-rc8

Steps to reproduce the problem:
Create a DO cluster.
Deploy workload .

Workload fails to start and is stuck in "Updating" state with error "Deployment does not have minimum availability."

see the following error in even logs:

```

Warning | FailedCreatePodSandBox | Failed create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "789cb3b7b05d970b835727769c4a6e33ad9472fe221c9351397df79468d2587b" network for pod "hello1-545b84875f-gj6mj": NetworkPlugin cni failed to set up pod "hello1-545b84875f-gj6mj_default" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted
-- | -- | --



```

**Comment by alena1108** [comment]
The fix is in `rancher/rancher:v2.1.7-rc9`

**Comment by sangeethah** [comment]
Tested with rancher server version - v2.1.7-rc9
Able to deploy workloads successfully.
